libr/include/r_core.h

```diff
@@ -924,6 +925,43 @@ R_API void r_core_anal_propagate_noreturn(RCore *core, ut64 addr);
 extern RCorePlugin r_core_plugin_java;
 extern RCorePlugin r_core_plugin_a2f;
 
+/* DECOMPILER PRINTING FUNCTIONS */
+/**
+* r_core_annotated_code_print_json() - Prints the data contained in RAnnotatedCode *code in JSON format.
+* @code: Pointer to a RAnnotatedCode
+*
+* Prints the data contained in RAnnotatedCode represented by the pointer 'code' in JSON format. 
+* The function will print the output in console using the function r_cons_printf();
+* 	
+* Return: Nothing
+*/
+R_API void r_core_annotated_code_print_json(RAnnotatedCode *code);
+/**
+* r_core_annotated_code_print() - Prints the decompiled code in the passed argument 'code'.
+* @code: Pointer to a RAnnotatedCode
+* @line_offsets: Pointer to a RVector that containes offsets for the decompiled code
+*
+* This function is used for the output of commands pdg and pdgo.
+* It can print the decompiled with offset and without offset. If line_offsets is a null pointer,
+* this function will print the output without offsets(pdg), otherwise the output will be
+* with offsets.
+* This function will print the output in console using the function r_cons_printf();
+* 
+* Return: Nothing
```

Comment:
```suggestion
* This function is used for printing the output of commands pdg and pdgo.
* It can print the decompiled code with or without offsets. If line_offsets is a null pointer,
* the output will be printed without offsets (pdg), otherwise, the output will be
* printed with offsets.
* This function will print the output in console using the function r_cons_printf();
* 
* Return: Nothing
```

Reply:
Changed as suggested.